<launch>
    <node name="statistical_removal" pkg="filters" type="statistical_removal" output="screen">
        <param name="subscribed_topic" type="string" value="/cloud_passed"/>
        <param name="published_topic" type="string" value="cloud_cleaned"/>
        <param name="meanK" type="int" value="20"/>
        <param name="mulThresh" type="double" value="1.25"/>
    </node>
</launch>
